npc/re/instances/octopus_cave.txt

```diff
@@ -43,63 +43,44 @@ mal_dun01,151,235,5	script	Starfish	4_ASTER,{
 
 	if (!.@party_id) {
 		mes("[Starfish]");
-		mes("You alone is powerless, hehe!");
-		mes("Better get someone to help you out.");
-		mes("Make a party, and come back later.");
+		mes("You alone is powerless, hehe! Better get someone to help you out. Make a party, and come back later.");
```

Comment:
Are these intended to be in one line? Is it desirable that the line break is not where the exclamation mark or the period are?